test/openshift/e2e/sequential/1-018_validate_disable_default_instance/03-errors.yaml

```diff
@@ -0,0 +1,295 @@
+  namespace: openshift-gitops
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+    argocds.argoproj.io/name: openshift-gitops
+    argocds.argoproj.io/namespace: openshift-gitops
+  labels:
+    app.kubernetes.io/managed-by: openshift-gitops
+    app.kubernetes.io/name: argocd-application-controller
+    app.kubernetes.io/part-of: argocd
+  name: openshift-gitops-argocd-application-controller
+  namespace: openshift-gitops
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+    argocds.argoproj.io/name: openshift-gitops
+    argocds.argoproj.io/namespace: openshift-gitops
+  labels:
+    app.kubernetes.io/managed-by: openshift-gitops
+    app.kubernetes.io/name: argocd-dex-server
+    app.kubernetes.io/part-of: argocd
+  name: openshift-gitops-argocd-dex-server
+  namespace: openshift-gitops
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+    argocds.argoproj.io/name: openshift-gitops
+    argocds.argoproj.io/namespace: openshift-gitops
+  labels:
+    app.kubernetes.io/managed-by: openshift-gitops
+    app.kubernetes.io/name: argocd-grafana
+    app.kubernetes.io/part-of: argocd
+  name: openshift-gitops-argocd-grafana
+  namespace: openshift-gitops
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  annotations:
+    argocds.argoproj.io/name: openshift-gitops
+    argocds.argoproj.io/namespace: openshift-gitops
+  labels:
+    app.kubernetes.io/managed-by: openshift-gitops
+    app.kubernetes.io/name: argocd-redis
+    app.kubernetes.io/part-of: argocd
+  name: openshift-gitops-argocd-redis
+  namespace: openshift-gitops
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+annotations:
+  argocds.argoproj.io/name: openshift-gitops
+  argocds.argoproj.io/namespace: openshift-gitops
+labels:
+  app.kubernetes.io/managed-by: openshift-gitops
+  app.kubernetes.io/name: argocd-redis-ha
+  app.kubernetes.io/part-of: argocd
+name: openshift-gitops-argocd-redis-ha
+namespace: openshift-gitops
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+annotations:
+  argocds.argoproj.io/name: openshift-gitops
+  argocds.argoproj.io/namespace: openshift-gitops
+labels:
+  app.kubernetes.io/managed-by: openshift-gitops
+  app.kubernetes.io/name: argocd-server
+  app.kubernetes.io/part-of: argocd
+name: openshift-gitops-argocd-server
+namespace: openshift-gitops
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: controller
+    app.kubernetes.io/managed-by: openshift-gitops
+    app.kubernetes.io/name: argocd-applicationset-controller
+    app.kubernetes.io/part-of: argocd-applicationset
+  name: openshift-gitops-applicationset-controller
+  namespace: openshift-gitops
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: "2023-05-31T02:59:31Z"
```

Comment:
@anandf Could you please remove this

Reply:
Yeah. missed it. Let me remove and run the E2E once to confirm.

Reply:
Removed the timestamp along with annotations and labels which are not required for the comparison.